pkg/system/phase4_configuring.go

```diff
@@ -1425,6 +1429,76 @@ func (r *Reconciler) ReconcileOBCStorageClass() error {
 	return nil
 }
 
+// ReconcileODFBucketLoggingPVC ensures the bucket logging PVC is properly configured for 'guaranteed' logging
+func (r *Reconciler) ReconcileODFBucketLoggingPVC() error {
+	log := r.Logger.WithField("func", "ReconcileBucketLoggingPVC")
+
+	// Return if Bucket logging type is either not specified or not set to 'guaranteed'
+	if r.NooBaa.Spec.BucketLogging.LoggingType != nbv1.BucketLoggingTypeGuaranteed {
+		log.Info("Bucket logging type is either not specified or not set to 'Guaranteed'. skipping ReconcileODFBucketloggingPVC")
+		return nil
+	}
+
+	// Return if bucket logging PVC already exist and support RWX access mode
+	if r.NooBaa.Spec.BucketLogging.BucketLoggingPVC != nil {
```

Comment:
Is there any reason all of these checks can't move to phase1_verifying?

Reply:
Moved it. Thanks